lib/ibmdb.js

```diff
@@ -621,9 +651,13 @@ IBMDB.prototype.createTable = function(model, cb) {
 IBMDB.prototype.dropTable = function(model, cb) {
   var self = this;
   var dropStmt = 'DROP TABLE ' + self.schema + '.' +
-    self.tableEscaped(model);
+                 self.tableEscaped(model);
+
```

Comment:
I would personally try to categorize related lines together according to our [Knowledge-base doc](https://github.com/strongloop/loopback-contributor-docs/blob/master/style-guide.md#line-spacing). So I would do it without empty line between lines which initialize variables. Please feel free to ignore it. Just a nitpick :-)